File: Code coach problems/Medium/C++/No_Numerals.cpp
#include <iostream>
#include <string>
using namespace std;
int main()
{
	string phrase;
	getline(cin, phrase);
	for (int i = 0; i < phrase.size(); i++)
	{
		if (phrase[i] >= '0' && phrase[i] <= '9')
		{
			switch (phrase[i])
			{
				case '0':
					cout << "zero";
					break;
				case '1':
					if(phrase[i+1] == '0')
					{
						cout << "ten";
						i++;
					}
					else
						cout << "one";
					break;
				case '2':
					cout << "two";
					break;
				case '3':
					cout << "three";
					break;
				case '4':
					cout << "four";
					break;
				case '5':
					cout << "five";
					break;
				case '6':
					cout << "six";
					break;
				case '7':
					cout << "seven";
					break;
				case '8':
					cout << "eight";
					break;
				case '9':
					cout << "nine";
					break;
			}
		}
		else 
			cout << phrase[i];
	}
}

File: Code coach problems/Medium/Python/No_Numerals.py
a = input()
res = ""
i = 0
while i != len(a):
	if a[i].isdigit():
		if a[i] == '0':
			res += 'zero'
		elif a[i] == '1':
			if i != len(a) -1 and a[i+1] == '0':
				res+="ten"
				i+=1
			else:
				res+="one"
		elif a[i] == '2':
			res+="two"
		elif a[i] == '3':
			res+="three"
		elif a[i] == '4':
			res+="four"
		elif a[i] == '5':
			res+="five"
		elif a[i] == '6':
			res+="six"
		elif a[i] == '7':
			res+="seven"
		elif a[i] == '8':
			res+="eight"
		else:
			res+="nine"
	else:
		res+=a[i]
	i+=1
	

print(res)
